deploy/laa-manage-your-civil-cases/values/uat.yaml

```diff
@@ -39,21 +39,15 @@ service:
   port: 80
 
 
-# ingress:
-#   hosts:
-#     - host: main-manage-your-civil-caases-uat.cloud-platform.service.justice.gov.uk
-#       secret: false
-#   annotations:
-#     external-dns.alpha.kubernetes.io/set-identifier: "manage-civil-cases-laa-manage-your-civil-cases-laa-manage-your-civil-cases-uat-green"
-#     external-dns.alpha.kubernetes.io/aws-weight: "100"
 ingress:
   enabled: true
   hosts:
-    - host: ""
+    - host: main-mcc-uat.cloud-platform.service.justice.gov.uk
```

Comment:
How come this isn't `manage-your-civil-cases-uat.cloud-platform.service.justice.gov.uk`?

Reply:
updated the name